projects/wave-core-new/src/lib/operators/operator-type.model.ts

```diff
@@ -0,0 +1,131 @@
+// export interface OperatorTypeConfig {} // tslint:disable-line:no-empty-interface
+
+/**
+ * Options allowed when cloning the operator
+ */
+export interface OperatorTypeCloneOptions {
+} // tslint:disable-line:no-empty-interface
+
+/**
+ * Dictionary for querying the server.
+ */
+export interface OperatorTypeMappingDict {
+} // tslint:disable-line:no-empty-interface
```

Comment:
Same here, should be removed.

Reply:
1187fb5a6148657ae9eb1e960c54ed287d841f02